### PR TITLE
Add more linux-api bits needed for epoll

### PR DESCRIPTION
**Pull request by robgjansen**

### CODE REVIEW

File: src/lib/linux-api/src/epoll.rs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+use crate::{bindings, const_conversions};
+use num_enum::{IntoPrimitive, TryFromPrimitive};
+
+bitflags::bitflags! {
+    /// Epoll create flags, as used with `epoll_create1`.
+    #[derive(Copy, Clone, Debug, Default, Eq, PartialEq)]
+    pub struct EpollCreateFlags: i32 {
+        const CLOEXEC = const_conversions::i32_from_u32(bindings::LINUX_EPOLL_CLOEXEC);
```
Comment (sporksmith) [inline src/lib/linux-api/src/epoll.rs:8]: We've generally been keeping the "type prefixes" for easier cross-referencing with man-pages etc; e.g. `EPOLL_CLOEXEC` here. I'm open to revisiting, but probably makes sense to be consistent one way or the other within the `linux-api` crate.

File: src/lib/linux-api/src/epoll.rs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+}
+
+/// Epoll control operation, as used with `epoll_ctl`.
+#[derive(Debug, Copy, Clone, Eq, PartialEq, IntoPrimitive, TryFromPrimitive)]
+#[repr(i32)]
+#[allow(non_camel_case_types)]
+pub enum EpollCtlOp {
+    ADD = const_conversions::i32_from_u32(bindings::LINUX_EPOLL_CTL_ADD),
```
Comment (sporksmith) [inline src/lib/linux-api/src/epoll.rs:17]: `EPOLL_CTL_ADD` etc

File: src/lib/linux-api/src/epoll.rs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    DEL = const_conversions::i32_from_u32(bindings::LINUX_EPOLL_CTL_DEL),
+}
+
+bitflags::bitflags! {
+    /// Epoll event types and input flags, which are ORed together in the `events` member of
+    /// `struct epoll_event`. As explained in `epoll_ctl(2)`, some flags represent event
+    /// types and other flags specify various input wakeup options.
+    #[derive(Copy, Clone, Debug, Default, Eq, PartialEq)]
+    pub struct EpollEvents: u32 {
+        const TYPE_IN = bindings::LINUX_EPOLLIN;
```
Comment (sporksmith) [inline src/lib/linux-api/src/epoll.rs:28]: `EPOLLIN` etc

**Review by sporksmith — APPROVED** [review]

File: src/lib/linux-api/src/epoll.rs (hunk 1 above)
Comment (robgjansen) [reply src/lib/linux-api/src/epoll.rs:8]: No problem. My fault for trying to be fancy :)

**Pull request merged**